### ISSUE TITLE
Beginners Guide: Play Clues (Question 1 & 2)

### ISSUE THREAD

**Issue by 0nen** [hanabi/hanabi.github.io#713]
https://hanabi.github.io/docs/beginner/play-clues-question-1

Q1:
> Bob marks down the identity of the card.
> What identity does Bob write down?

Q2:
> Bob marks down the identity of the card.
> What identity does Bob write down? (Be specific.)

Up to this point there has been no discussion of how you "mark" cards. I suggest rewording the question to simply ask what Bob should think the card is based on the clue.

Edit:

This affects other questions as well, I just haven't read through the whole beginners guide, for example https://hanabi.github.io/docs/beginner/delayed-play-clues-question-1 has the same issue. There are probably more.

Another option is to introduce the concept of marking cards early on.

https://hanabi.github.io/docs/beginner/critical-save-question-1 
On this question, it asks
> What identity does Bob write on the card?

Similar issue here, inconsistent wording (variations used are mark down, write down, write on... maybe more)
https://hanabi.github.io/docs/beginner/critical-save-question-2
> Cathy would mark her slot 4 card as either:

I think being consistent in the wording would be good as well (if the wording is kept). e.g. change the ones that say "What identity does Bob write down?" to be "What identity does Bob write on the card?" because that is more direct.

**Comment by Zamiell** [comment]
When you play Hanabi, you will **need** to write *card notes* and this page explains why.

<br />

## Explicit and Implicit Information

- In Hanabi, players will know *explicit* information and *implicit* information about the cards in their hand:
  - *Explicit* information is from the clues "on" the card.
  - *Implicit* information is from the context of the game *when* the card was clued.
- For example, in a 3-player game:
  - It is the first turn of the game and nothing is played on the stacks.
  - Alice clues Bob red, touching a card on his slot 1. Alice intends for this to be a *Play Clue*, signaling to Bob that he has a red 1 and that he should play it on the stacks right now.
  - From Bob's perspective, he knows that *explicitly*, his red card can be red 1, red 2, red 3, red 4, or red 5.
  - But Bob also knows that *implicitly*, Alice intends for him to play this red card, so it must be exactly the red 1.

<br />

## Players Need to Track Implicit Information

- On [Hanab Live](https://hanab.live/), the interface automatically will keep track of the clues that touch the cards. (And if you are playing Hanabi in real life, you could accomplish this by using counters.)
- However, this automated tracking of *explicit* information is not sufficient, because there is so much more *implicit* information than *explicit* information.
- In Hanabi games where players only give basic *Play Clues* to each other (like in the previous example), remembering all of the *implicit* information about the cards is trivial. But as games get more and more complicated, remembering *implicit* information about every card in the game is impossible.
- Thus, in order to play Hanabi properly, players **must** go above and beyond the automated *explicit* tracking and record all of the implicit information themselves.

<br />

## Writing Notes

- Handily, Hanab Live allows players to record *implicit* information on a card by right-clicking on it. This pops up a text box, allowing players to record any information that they want. This is card a *card note*.
- **Players should record a note on the affected card after every single clue.**
- In the previous example, Bob should right-click on his red card and write "red 1", so that he doesn't forget for later.
- Furthermore, both Alice and Cathy **also** right-click on Bob's red and write "red 1", so that they can keep track of what Bob thinks about the card. In this way, **everybody writes notes on everybody's cards.**

<br />

## Avoiding Mistakes

- Since the red clue above was so simple, you may think that writing a note of "red 1" is silly and a waste of time. However, realize that Bob **might not play the red 1 for many, many turns**, at which point the game state may have dramatically changed.
- The most common beginner mistake that we see is failing to write a note on a card, and then forgetting some piece of information later on in the game.
- Do not forget to do this! Write *card notes* **every single time**!

**Comment by vEnhance** [comment]
+1 to this --- I was actually thinking to open an issue myself saying that we should state up-front that notes are really important.

**Comment by vEnhance** [comment]
Might also want to add something about how writing "r1" or "red 1" will cause the card image to "snap", so to speak --- it's a bit of a surprise the first time you see it.

In the section on finesses, we might also mention the special note "[f]" on the website (and perhaps similarly for cm later on?).

**Comment by Zamiell** [comment]
i'd rather omit that information to keep it more concise

**Comment by vEnhance** [comment]
how about in level 2 where reverse finesses are introduced

**Comment by Zamiell** [comment]
how about a new section in level 1 called "notes" that describes:
1) how a card will snap after writing an identity note
2) how "f" is used to mark cards as finssed
3) how "cm" is used to mark cards are chop moved, with the caveat of `(*Chop Moved* cards are special cards that are introduced at a later level.)`

**Comment by vEnhance** [comment]
That sounds good. Would this be the same page as the one you put in the first comment?

**Comment by Zamiell** [comment]
my previous comment is a new section in the beginners guide, which will be pushed to live momentarily

**Comment by Zamiell** [comment]
onen i made some changes to the beginners guide today and i believe it is now consistent, open a new issue if you find anything else

evan do you want to submit a PR for that?

will close this issue now

**Comment by 0nen** [comment]
https://hanabi.github.io/docs/beginner/save-principle-question-2/

> If Alice should give a clue, what identity will be written on the focused card? (Be specific.)

This one was accidentally missed for updating the "card notes" text

**Comment by Zamiell** [comment]
fixed thx

**Comment by vEnhance** [comment]
Yep, I’ll submit a PR today or tomorrow on level 1 page.--
*Evan Chen (陳誼廷) *[via evanchen at mit.edu]
https://web.evanchen.cc/ || https://www.facebook.com/evanchenmath/
